src/main/groovy/com/google/protobuf/gradle/GenerateProtoTask.groovy

```diff
@@ -166,6 +197,14 @@ public class GenerateProtoTask extends DefaultTask {
     state = State.FINALIZED
   }
 
+  String getDescriptorPath() {
+    if (!generateDescriptorSet) {
+      return null
```

Comment:
I would throw an `IllegalStateException` because `getDescriptorPath()` is not supposed to be called if `generateDescriptorSet==null`.